### ISSUE TITLE
Error: Upgrade de.systopia.donrec to revision 0140

### ISSUE THREAD

**Issue by systopia** [systopia/de.systopia.donrec#74]
I get the following error during the extension upgrade process to ``1.5``
```
[Error: Upgrade de.systopia.donrec to revision 0140]
donrec_default_profile,donrec_packet_size,donrec_pdfinfo_path not valid settings
```

**Comment by systopia** [comment]
The underlying issue is this: the settings definition in question have already been removed in ``1.5``, but were needed in ``1.4``. So when upgrading from ``1.3`` to ``1.5`` or higher on a recent CiviCRM version, the settings API will throw that error.

The fix uses the internal functions of ``CRM_Core_BAO_Setting`` instead of the API, which don't have this restriction.

**Comment by systopia** [comment]
fixed with ``1.6``
